### PR TITLE
CI: run sniffs and fix codebase

### PR DESCRIPTION
**Pull request by DFoxinator**
@be-dmitry 
@ingluisjimenez

### CODE REVIEW

**Comment by be-dmitry** [comment]
👍

File: src/RulesetProviders/ConfigProvider.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ public function __construct( array $config ) {
 
     $this->_config = $config;
 
-  }
+  } // __construct
```
Comment (ingluisjimenez) [inline src/RulesetProviders/ConfigProvider.php]: need docblocks for the constructor and the `private $_config` above

File: src/RulesetProviders/ConfigProvider.php [hunk 2]
```diff
@@ -32,23 +32,23 @@ public function getRuleset( $feature ) {
...
 
-    $this->_generated_rulesets[$feature] = $ruleset;
+    $this->_generated_rulesets[ $feature ] = $ruleset;
 
     return $ruleset;
 
   } // getRuleset
 
-}
+} // ConfigProvider
```
Comment (ingluisjimenez) [inline src/RulesetProviders/ConfigProvider.php]: also, this guy maybe could be named `ConfigRulesetProvider` ?

**Comment by ingluisjimenez** [comment]
👍

File: src/RulesetProviders/ConfigProvider.php (hunk 1 above)
Comment (DFoxinator) [inline src/RulesetProviders/ConfigProvider.php]: added

File: src/RulesetProviders/ConfigProvider.php (hunk 2 above)
Comment (DFoxinator) [inline src/RulesetProviders/ConfigProvider.php]: changed

**Pull request merged**